classifier.ipynb

```diff
@@ -39,6 +40,86 @@
    "source": [
     "DX.shape, DY.shape, DZ.shape"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# random forest\n",
+    "from sklearn.ensemble import RandomForestClassifier\n",
+    "from sklearn.model_selection import cross_val_score\n",
+    "clf = RandomForestClassifier(n_estimators=200, max_depth=15, random_state=0)\n",
+    "clf.fit(DX, DZ)\n",
+    "print(clf.feature_importances_)\n",
+    "scores = cross_val_score(clf, DX, DZ, cv=10)\n",
+    "print(scores)"
```

Comment:
What about putting three classifiers' scores in one dataframe and show it in notebook?

Reply:
We mentioned in the meeting, that instead of 10 numbers, we want 1 number. Just to clarify, the 1 number is the accuracy: the correct classification /  the total number of classification. So that each setup (ways of classification + hyperparameter) have a single accuracy.

Reply:
Thanks for reminding me this - if so, should `score` instead of `scores` a better name. Also, please include the output of the running inside the notebook as well - this is why notebook is so useful to us :)

Reply:
<img width="613" alt="Screen Shot 2019-07-28 at 7 00 24 PM" src="https://user-images.githubusercontent.com/13213722/62017040-f9138800-b169-11e9-9496-6bb33b940cd8.png">

Reply:
The above is obtained by cross_val_predict and accuracy score.